l10n_be_national_number/models/res_partner.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2009-2019 Noviat.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    national_number = fields.Char(string="National Number")
```

Comment:
Is this field even used? `id_numbers` appears to be the field that is _actually_ used. This field `national_number` is displayed in no view. I also can't find downstream usage of this field.

Reply:
I think you are right ! I think I made a mistake.

Reply:
Maybe let's delete this module then ?

Reply:
The only important thing in this module is the category definition in the data file :) We can remove this Python code.